tests/test_arcaflow_plugin_stressng.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 
 import unittest
-import yaml
 import math
+import yaml
 import stressng_schema
 import stressng_plugin
 from arcaflow_plugin_sdk import plugin
```

Comment:
Consider feeding this to `isort` -- I think it will have a third opinion on the ordering, here.

Reply:
The ordering was the suggestion of Pylint. It's not something I get worked up about, personally.